sdk/servicebus/service-bus/tests.yml

```diff
@@ -24,7 +24,11 @@ jobs:
           OSVmImage: "windows-2019"
           TestType: "browser"
       EnvVars:
-        AAD_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-        AAD_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-        AAD_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
-        SERVICEBUS_CONNECTION_STRING: $(SERVICEBUS_CONNECTION_STRING)
+        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+        AZURE_SUBSCRIPTION_ID: $(test-subscription-id)
+        CLEAN_NAMESPACE: "true"
+        RESOURCE_GROUP: $(service-bus-test-resource-group)
+        SERVICEBUS_CONNECTION_STRING: $(service-bus-test-connection-string)
+        SERVICEBUS_CONNECTION_STRING_BROWSER: $(service-bus-test-connection-string-browser)
```

Comment:
The below are not needed here
- CLEAN_NAMESPACE
- RESOURCE_GROUP
- SERVICEBUS_CONNECTION_STRING_BROWSER